sterling/tests/test_sterling.py

```diff
@@ -18,15 +18,16 @@
     "ipval": str(fake)
 }
 
+
 @patch.object(Session, 'request')
 def test_enquiry(mock_get):
     data = responses["enquiry"]
     mock_get.return_value = R(responses["enquiry"])
-    assert Enquiry(header).name_enquiry(query) == data, "should return an object"
+    assert Enquiry(header).InterbankNameEnquiry(query) == data, "should return an object"
 
 
 @patch.object(Session, 'request')
 def test_transfer(mock_post):
     data = responses["transfer"]
     mock_post.return_value = R(responses["transfer"])
-    assert Transfer(header).transfer(body) == data, "should return an object"
+    assert Transfer(header).InterbankTransferReq(body) == data, "should return an object"
```

Comment:
Change `assert Transfer(header).InterbankTransferReq(body) == data, "should return an object"`
To `assert Account(header).InterbankTransferReq(body) == data, "should return an object"`

Basically, rename Transfer to Account here and anywhere else applicable.